symplyphysics/laws/kinematic/vector/velocity_relative_to_reference_frame.py

```diff
@@ -0,0 +1,66 @@
+from sympy import Expr, symbols
+from symplyphysics import (
+    units,
+    validate_input,
+    validate_output,
+    Vector,
+    Quantity,
+    QuantityVector,
+    add_cartesian_vectors,
+    scale_vector,
+)
+from symplyphysics.core.vectors.arithmetics import (
+    diff_cartesian_vector_components,
+    integrate_cartesian_vector_components,
+)
+
+# Description
+## For any reference frame, whether it is inertial or not, the motion relative to it can be described using
+## the position vector relative to that frame's origin.
+
+# Law: v_rel = (dx/dt, dy/dt, dz/dt)
+## v_rel - vector of velocity relative to reference frame S
+## r = (x, y, z) - vector of position relative to frame S
+## d/dt - time derivative
+
+
+def relative_velocity_law(
+    position_: Vector,
+    time_: Expr,
+) -> Vector:
+    return diff_cartesian_vector_components(position_, time_)
```

Comment:
I've thought about it, but I guessed it's less convenient for the user. I would expect them to have their own time symbol and they would explicitly provide it here as an argument. Otherwise they would need to first `.subs` their symbol with the one in this law, perform the calculation, and then `.subs` it back to their symbol again.